### PR TITLE
Add force enable command line flag for Telemetry

### PR DESCRIPTION
**Pull request by joeltine**
b/286091096

Change-Id: I7c96de962bab0f2298141e1212f5af82caa54fa6

### CODE REVIEW

File: cobalt/browser/metrics/cobalt_metrics_services_manager_client.cc [hunk 1]
```diff
@@ -57,13 +59,14 @@ bool CobaltMetricsServicesManagerClient::IsMetricsConsentGiven() {
   return enabled_state_provider_->IsConsentGiven();
 }
 
-// If the user has forced metrics collection on via the override flag.
+// If the user has forced metrics collection on via the override flag. This
+// being set trumps/overrides any other mechanism to enable telemetry (e.g.,
```
Comment (TyHolc) [inline cobalt/browser/metrics/cobalt_metrics_services_manager_client.cc:63]: nit: Maybe reword to `This switch being set...` or just `This switch trumps/overrides...`

**Review by TyHolc — APPROVED** [review]

File: cobalt/browser/metrics/cobalt_metrics_services_manager_client.cc (hunk 1 above)
Comment (joeltine) [reply cobalt/browser/metrics/cobalt_metrics_services_manager_client.cc]: Done.
